sdk/ledger-service/http-json/src/main/scala/com/digitalasset/http/PackageService.scala

```diff
@@ -375,28 +375,53 @@ object PackageService {
   final case class ContractTypeIdMap[CtId[T] <: ContractTypeId[T]](
       all: Map[RequiredPkg[CtId], ResolvedOf[CtId]],
       unique: Map[NoPkg[CtId], ResolvedOf[CtId]],
-      nameIds: Map[Ref.PackageName, String], // TODO(ETX-451): Multiple package ids per package name
+      nameIds: Map[Ref.PackageName, NonEmpty[Seq[String]]],
```

Comment:
Both `nameIds` and `idNames` are keyed off package-name so they could be combined into a single map.